src/Template/ClientMethodTemplate.cs

```diff
@@ -316,7 +316,7 @@ public void Write(ClientMethod clientMethod, JavaType typeBlock)
                         function.Indent(() =>
                         {
                             function.Text(".map(");
-                            function.Lambda(returnValueTypeArgumentType.ToString(), "res", "res.body()");
+                            function.Lambda(returnValueTypeArgumentType.ToString(), "res", "res.value()");
```

Comment:
This and many other simple wording changes are to accommodate the recent changes in azure-common.